package.json

```diff
@@ -16,6 +16,8 @@
     "lint-fix": "yarn sfdx-lint --fix",
     "postcompile": "tsc -p test; tsc -p typedocExamples",
     "prepack": "yarn sfdx-build",
+    "prepare": "yarn run snyk-protect",
+    "snyk-protect": "snyk protect",
```

Comment:
`snyk protect` changes dependencies right? I want CI to run on any new deps.

Reply:
It does not appear that `snyk protect` alters the dependency tree - only that it will apply patches that are known to snyk

<https://support.snyk.io/hc/en-us/articles/360000920818-What-is-the-difference-between-snyk-test-protect-and-monitor->
```
snyk protect is used to apply patches to your vulnerable dependencies.
It's useful after opening a fix pull request from our website (GitHub only)
or after running snyk wizard on the CLI. snyk protect reads a .snyk policy
file to determine what patches to apply.
```

None of those dependencies (transitive or direct) in the `resolutions` section had patches on the snyk website, but did have updated versions available.

What I'm not sure of is whether a downstream project like the sf-fx-sdk-nodejs or its resulting Functions will respect the `resolutions` in this project or if we need to repeat them in downstream projects.  Not sure how to test that without finding a way to do a local release of this project to pull in as a dependency -- any ideas?